### ISSUE TITLE
Keyboard navigation on dropdowns does not work?

### ISSUE THREAD

**Issue by christianalfoni** [silviomoreto/bootstrap-select#902]
Hi,

Keyboard navigation does not work on dropdowns. Tested on Chrome both locally and on the react-bootstrap site. I saw an earlier issue on this, maybe it has broken again?

Thanks for making this port, its awesome!

**Comment by christianalfoni** [comment]
Ops, wrong repo
